docs/pages/The-Voxblox-Node.rst

```diff
@@ -231,15 +234,14 @@ Output Parameters
   The method that will be used for coloring the mesh. Options are "color", "height", "normals", "lambert" and "gray".
 ``mesh_min_weight`` `1e-4`
   The minimum weighting needed for a point to be included in the mesh.
-``update_mesh_every_n_sec`` `0.0`
-  Rate at which the mesh topic will be published to, a value of 0 disables. Note, this will not trigger any other mesh operations, such as generating a ply file.
 ``publish_tsdf_map`` `false`
   Whether to publish the complete TSDF map periodically over ROS topics.
 ``publish_esdf_map`` `false`
   Whether to publish the complete ESDF map periodically over ROS topics.
-``publish_tsdf_info`` `false`
-  Enables publishing of ``tsdf_pointcloud``, ``surface_pointcloud`` and ``occupied_nodes``.
-``publish_pointclouds`` | If true the tsdf and esdf (if generated) is published as a pointcloud when the mesh is updated.
+``publish_pointclouds`` `false`
+  If true the tsdf and esdf (if generated) is published as a pointcloud when the mesh is updated or whenever the map is updated if `publish_pointclouds_on_update` is set to true as well.
```

Comment:
No sorry, bad description. 2 options:
If `publish_pointclouds_on_update`: every time a new *input* pointcloud is received, re-publish visualization pointclouds.
If NOT `publish_pointclouds_on_update` but `publish_pointclouds`: just do the pointcloud publishing when the mesh is updated (since that's how often you change the visualization). I'll change the description to be clearer.